emulsify-ds/compound
# feat: Pager component: adding first and last and adding addAttributes

**Summary**

This PR improves the Pager component for Compound.

- Adds First and Last option to Pager (this is regularly seen in Drupal sites)
- Adds `add_attributes` to template (to make it easier to extend them when we implement them in other components)
- Renamed `pager-ellipses.yml` to `pager-next-ellipses.yml` to match `prev` example for consistency
- Hid the ellipses on the basic pager example (it didn't seem to make sense to keep them in without the arrows there too.)

**How to review this PR**
- Check out `feat-pager` branch
- `npm run develop`
- Navigate to:
  http://localhost:6006/?path=/story/molecules-menus-pager--basic
  http://localhost:6006/?path=/story/molecules-menus-pager--with-next
  http://localhost:6006/?path=/story/molecules-menus-pager--with-both
  http://localhost:6006/?path=/story/molecules-menus-pager--with-previous
  http://localhost:6006/?path=/story/molecules-menus-pager--with-first-and-last
- Verify that changes above make sense and the code looks good
